netlify/edge-functions/redirect-to-latest.ts

```diff
@@ -0,0 +1,41 @@
+export default async (request: Request, context) => {
+  const url = new URL(request.url);
+  let pathname = url.pathname;
+  // Normalize pathname by removing consecutive slashes
+  pathname = pathname.replace(/\/\/+/g, '/');
+  // Match version paths like /vX.XX/
+  const versionMatch = pathname.match(/^\/v\d+\.\d+/);
+
+  if (versionMatch) {
+    // Define the main section paths that should be redirected
+    const redirectableSections = ['/', '/about', '/blog', '/get-involved', '/news', '/search'];
+    const versionPath = versionMatch[0]; // ex /v1.15
+    const remainder = pathname.slice(versionPath.length) || '/';
+    // Check if the remainder starts with an optional leading slash followed by a language code
+    const languageCodeMatch = remainder.match(/^\/?([a-z]{2})(\/|$)/);
+    let languageCode: string | null = null;
+    let remainingPathAfterLang = remainder;
+
+    if (languageCodeMatch) {
+      languageCode = languageCodeMatch[1];
+      remainingPathAfterLang = remainder.slice(languageCodeMatch[0].length);
```

Comment:
...but what you end up with here is `blog`, not `/blog`, thus it does not match.